packages/redux-routine/src/rungen.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'rungen' {
```

Comment:
@youknowriad Would you be interested in publishing types for `rungen`? We could include the types in the package without requiring any code changes.

Reply:
Whatever works best for you. I'm also happy to add collaborators on that package, I didn't touch it for some time now 😬

Reply:
Thanks. Providing the types from the library itself seems ideal to me, although we don't need to wait for it here.